tests/external/reproductions.repos.yaml

```diff
@@ -30,3 +30,7 @@
   url: https://github.com/QuantStack/xeus-cling
   ref: 0.4.5
   verify: jupyter kernelspec list
+- name: Test compiling R package
```

Comment:
Can we put a test under https://github.com/jupyter/repo2docker/tree/master/tests/r as well?

External repos for tests are (like other external things) one of the reasons we get flaky tests so where ever possible I think we should have a test in this repository. So when the travis network connection is flaky we don't get a red test.

Reply:
Sure, I can put the content of the external repo into the `R` test folder for that case? And do you mean to just replace the external test in that case?

Reply:
Jupp.

I'd copy what the `simple/` subdirectory is doing and come up with a nice descriptive name :)